amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/persistence/SqlSessionFactoryProvider.java

```diff
@@ -123,6 +126,11 @@ public void init(Configurations config) {
    * @param config
    */
   private void createTablesIfNeed(Configurations config) {
+    boolean initSchema = config.getBoolean(AmoroManagementConf.DB_INIT_SCHEMA);
+    if (!initSchema) {
+      LOG.info("Skip database schema initialize.");
```

Comment:
I recommend adding more log outputs for each situation here: 

- the table was created automatically
- the table already exists so there is no need to create it automatically
- automatic creation is not enabled, so the table was not created automatically.

Reply:
fixed